waspc/src/Generator/Generators.hs

```diff
@@ -66,6 +66,30 @@ generateSrcDir wasp
     ++ EntityGenerator.generateEntities wasp
     ++ Generator.Button.generateButtons wasp
     ++ [generateReducersJs wasp]
+    -- NOTE(matija): for demo only.
+    ++ maybeGenerateXRayDir (getIsXRayModeEnabled wasp)
+
+maybeGenerateXRayDir :: Bool -> [FileDraft]
```

Comment:
You could separate this generator, for xray, into separate file maybe.

Reply:
Good idea, separated it.